traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -19,89 +19,87 @@
 import java.net.URL;
 import java.util.*;
 
-
 public class Router {
     private static final String DEFAULT_ENDPOINT = "https://traveler.rc.guestlogix.io/v1";
     private static String TAG = "Traveler";
```

Comment:
Maybe change this TAG as well while youre at it

Reply:
removed